platform/darwin/src/MGLStyle.h

```diff
@@ -270,6 +270,13 @@ MGL_EXPORT
  */
 @property (nonatomic) MGLTransition transition;
 
+/**
+ A boolean value indicating whether label placement transitions are enabled.
+ 
+ The default value of this property is `YES`.
+ */
+@property (nonatomic, assign) BOOL enablePlacementTransitions;
```

Comment:
This property has an imperative name, which makes it look like an action method in Objective-C:

```objc
// Initial state: placement transitions are disabled.
[style enablePlacementTransitions];
// Expected behavior: placement transitions are enabled.
// Actual behavior: nothing changed; return value discarded.
```

This property should be renamed to `placementTransitionsEnabled` or preferably `performsPlacementTransitions`. (`enablesPlacementTransitions` is problematic because it’s the developer, not the style, that enables them.)

Reply:
#13604